node/sc/sub_bridge_handler.go

```diff
@@ -167,11 +174,33 @@ func (sbh *SubBridgeHandler) getRemoteGasPrice() uint64 {
 	return sbh.remoteGasPrice
 }
 
+// setRemoteGasPrice sets parent chain's gasprice with upperboundbasefee
```

Comment:
```suggestion
// setRemoteGasPrice sets parent chain's gasprice
```
I think still `setRemoteGasPrice` is used for setting gas price of the parent chain.

Reply:
Yes. The comment was for under KIP71. Can I ask why the suggestion removes `setting the upperboundbasefee`?

Reply:
Is this called only under KIP-71 hard fork?

Reply:
Oops :( I misread the function as `setRemoteChainValues`. Thank you.